### PR TITLE
[TSK-1338] Update issue filters order

### PR DESCRIPTION
**Pull request by s0gorelkov**
# Contribution checklist
Also fixed a bug with duplicated filters for collection attributes such as "labels"

Screenshot 2023-05-04 at 17 54 09

## Checklist

* [ ] - UI test added to added/changed functionality?
* [x] - Screenshot is added to PR if applicable ?
* [x] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [x] - Does the translations are up to date?
* [x] - Does it well tested?
* [x] - Tested for Chrome.
* [ ] - Tested for Safari.
* [x] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [x] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [x] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

[TSK-1338](https://front.hc.engineering/workbench/platform/tracker/TSK-1338)

### CODE REVIEW

File: models/tracker/src/index.ts [hunk 1]
```diff
@@ -1350,7 +1350,21 @@ export function createModel (builder: Builder): void {
...
+      'labels',
+      'title',
+      'sprint',
+      'component',
+      'dueDate',
+      'createOn',
+      'modifiedOn',
+      'modifiedBy'
+    ],
+    ignoreKeys: ['number', 'estimation', 'attachedTo', 'space']
```
Comment (BykhovDenis) [inline models/tracker/src/index.ts]: You don't need ignore hidden fields. Also 'space' filter here is good especially for 'my issues' mode

File: plugins/view-resources/src/components/filter/FilterTypePopup.svelte [hunk 2]
```diff
@@ -85,6 +85,18 @@
     mixin: ClassFilters
   ): void {
     const ignoreKeys = new Set(mixin.ignoreKeys ?? [])
+
+    if (mixin.strict) {
```
Comment (BykhovDenis) [inline plugins/view-resources/src/components/filter/FilterTypePopup.svelte]: When is strict we should don't try build keys from hierarchy (desc and asc mixins)

**Review by BykhovDenis — APPROVED** [review]

File: plugins/view-resources/src/components/filter/FilterTypePopup.svelte (hunk 2 above)
Comment (s0gorelkov) [reply plugins/view-resources/src/components/filter/FilterTypePopup.svelte]: Done.

File: models/tracker/src/index.ts (hunk 1 above)
Comment (s0gorelkov) [reply models/tracker/src/index.ts]: There are no hidden fields I think. If I remove any key from this array, the filter appears in the filter list.
Thanks for note about "space" - removed.

**Review by haiodo — APPROVED** [review]

**Pull request merged**